.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/ruby:latest
+        steps:
+            - checkout
+            - run:
+                name: Which ruby?
+                command: ruby -v
+            - run:
+                name: Which bundler?
+                command: bundle -v
+            - run:
+                name: Bundle Install
+                command: bundle check || bundle install
```

Comment:
Is `bundle check ||` really necessary? I may be wrong but I think `bundle install` is a no-op if the bundle is already up-to-date, so just `bundle install` could be enough there?

Reply:
I found interesting things about this command. Initialy I copy paste it from CircleCi config for ruby project. But when you commented this line I searched explanation on bundler. I found this issue [#95](https://github.com/capistrano/bundler/issues/95). related to this line [bundler.cap](https://github.com/capistrano/bundler/blob/2618cde82352e845a84e20a095b5b043e96f2781/lib/capistrano/tasks/bundler.cap#L31)

So when `bundle install` is launched a test with `bundle check` is executed.

So I will delete `bundle check` command.